src/foam/nanos/google/api/sheets/import/GoogleSheetsImportConfig.js

```diff
@@ -0,0 +1,110 @@
+foam.CLASS({
+  package: 'foam.nanos.google.api.sheets',
+  name: 'GoogleSheetsImportConfig',
+  documentation: 'GoogleSheetsImportConfig contains info about import',
+  properties: [
+    {
+      name: 'importClassInfo',
+      class: 'Class',
+      javaType: 'foam.core.ClassInfo',
+      hidden: true,
+    },
+    {
+      name: 'googleSheetLink',
+      class: 'String',
+      view: {
```

Comment:
Here and elsewhere, you can just do: view: 'foam.u2.TextField'.

Reply:
fixed